package.json

```diff
@@ -20,18 +20,19 @@
   "scripts": {
     "prebuild": "npm run -s install-if-deps-outdated && php ./bin/update-version.php",
     "build": "npm run build:feature-config && npm run build:packages && cross-env NODE_ENV=production webpack",
-    "postbuild": "npm run -s i18n:php && npm run -s i18n:pot",
+    "postbuild": "npm run -s i18n:pot && npm run -s i18n:build",
     "build:feature-config": "php bin/generate-feature-config.php",
     "build:packages": "node ./bin/packages/build.js",
     "build:release": "./bin/build-plugin-zip.sh",
     "clean": "rimraf ./dist ./packages/*/build ./packages/*/build-module ./packages/*/build-style",
     "predev": "npm run -s install-if-deps-outdated && php ./bin/update-version.php",
     "dev": "cross-env WC_ADMIN_PHASE=development npm run build:feature-config && cross-env WC_ADMIN_PHASE=development npm run build:packages && cross-env WC_ADMIN_PHASE=development webpack",
     "docs": "node ./bin/generate-docs",
-    "i18n": "npm run -s i18n:js && npm run -s i18n:php && npm run -s i18n:check && npm run -s i18n:pot",
+    "i18n": "npm run -s i18n:js && npm run -s i18n:check && npm run -s i18n:pot && npm run -s i18n:build",
+    "i18n:build": "php bin/combine-pot-files.php languages/woocommerce-admin.po languages/woocommerce-admin.pot",
```

Comment:
I'm not quite following on why the po and pot files need to be combined here.  Could you give me a quick rundown of why we do this?

Reply:
> Could you give me a quick rundown of why we do this?

Each tool generates a complete file including a header. If the same output file name is used, both tools overwrite an existing file. Some strings exist in both files. If you switch to master and run `npm run i18n` then edit the .pot file you'll see references like `#: languages/woocommerce-admin.php:123`. Those originate from the `npm run i18n:js`.

We could just remove one of the headers and concatenate the files. But, if we do that then any string that is in both JS & PHP appears in the file for translation twice.

Reply:
That makes sense.  Out of curiosity, how will translations be managed going forward?  It seems like any changes to strings will require a new po file and any previous translations will be overwritten.  But maybe this is best left to translation editors like Poedit.